stadtmeldeapp/src/main/java/com/stadtmeldeapp/Controller/CategoryController.java

```diff
@@ -36,8 +42,12 @@ public ResponseEntity<MaincategoryEntity> getMainCategoryById(@PathVariable("id"
 
     @PostMapping("/main")
     public ResponseEntity<MaincategoryEntity> createOrUpdateMainCategory(
-            @RequestBody @NonNull MaincategoryEntity mainCategory) {
-        MaincategoryEntity savedMainCategory = categoryService.saveMainCategory(mainCategory);
+            @RequestBody @NonNull MaincategoryDTO mainCategoryDTO) {
```

Comment:
Könntest du die Logik vielleicht in den Service packen? Aktuell dient dieser eigentlich nur zur Weiterleitung vom Repository, sprich wir könnten ihn auch weg lassen.